### DISCUSSION TITLE
Disable "Selected data has been copied to the clipboard." notification.

### DISCUSSION THREAD

**Discussion by Lysunder** [microsoft/azuredatastudio#24148] [Q&A]
Is there a way to disable the "Selected data has been copied to the clipboard." notification? I didn't find a setting for it.

**Comment by erinstellato-ms** [comment]
In the Insider build, we've added a setting to disable it (Query Editor > Results : Show Copy Completed Notification).  This will be available in the next stable release (original issue here: https://github.com/microsoft/azuredatastudio/issues/24009).
